Yoast/javascript
# Change desktop font size to match google

## Summary


This PR can be summarized in the following changelog entry:

* Changes desktop snippet preview to match Google's new font sizes.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Inspect the snippet preview. Make sure the mobile snippet preview fot sizes haven't changed, and the desktop snippet preview font sizes are the following:
Title: 20px
URL: 16px
Meta description: 14px

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  * Snippet preview

## UI changes

## Quality assurance


Fixes Yoast/wordpress-seo#13421
